src/frameName.cpp

```diff
@@ -135,13 +136,16 @@ char* FrameName::truncate(char* name, int max_length) {
 
 const char* FrameName::decodeNativeSymbol(const char* name) {
     const char* lib_name = (_style & STYLE_LIB_NAMES) ? Profiler::instance()->getLibraryName(name) : NULL;
+    bool report_lib_name_from_symbol = lib_name == NULL && strcmp(name, "call_stub") != 0 &&
```

Comment:
`call_stub` specifically? Why?

Reply:
I want to avoid unnecessary calls to `findLibraryByName()`. `call_stub` is a synthetic symbol that is often an argument to `decodeNativeSymbol()`.

Reply:
I'm pretty sure there are other synthetic symbols, that's why I'm asking. So in any case, `findLibraryByName` will fail for this name, and the check is only a "hotpath" for this common case?

If so, please document that in the code.

Reply:
Kk :)